object_store/src/gcp.rs

```diff
@@ -575,6 +583,18 @@ mod test {
         service_account: String,
     }
 
+    impl GoogleCloudConfig {
+        fn build(self) -> Result<GoogleCloudStorage> {
```

Comment:
```suggestion
        fn build_test(self) -> Result<GoogleCloudStorage> {
```
Perhaps something to prevent it accidentally getting refactored to be public or something